### PR TITLE
Update package.json http2-wrapper

### PR DESCRIPTION
**Pull request by masx200**

### CODE REVIEW

**Comment by ronag** [comment]
I'm not entirely comfortable with this library. Maybe you could instead update the README with how http2-proxy and http2-wrapper could be used together?

File: package.json [hunk 1]
```diff
@@ -1,4 +1,10 @@
 {
+
+
+"dependencies": {
+		"http2-wrapper": "1.0.0-beta.4.4"
```
Comment (ronag) [inline package.json:5]: I would rather not add this dependency and instead let the user choose their library of choice for this. I wouldn't mind adding an example in the README of how to do this.

Comment (masx200) [reply package.json:5]: peerDependencies  http2-wrapper

**Comment by ronag** [comment]
Fixed in https://github.com/nxtedition/node-http2-proxy/commit/e0b93240b40bb4720c9dc903e6d6736df74d3fdb

**Pull request closed without merging**